app/controllers/api/oembed_controller.rb

```diff
@@ -14,8 +14,20 @@ def show
   def stream_entry_from_url(url)
     params = Rails.application.routes.recognize_path(url)
```

Comment:
I don't think this change needs to hold back merging this ... but it might be good to use a better local variable name here, since this is happening inside of a controller, and `params` is already a method (and used up in the `show` action...)